webpack.dev.js

```diff
@@ -3,10 +3,13 @@ const { targets, createConfig } = require('./webpack.common');
 const path = require('path');
 const PORT = 7936;
 
+const katexTarget = targets.shift();
+katexTarget.entry = './static/main.js';
+
```

Comment:
It would be nice to have a single entry point, but it would increase the unminified distribution file(`katex.js`) by 2-3 KB.

Reply:
@kevinbarabash Yes, but then webpack will rebuild both bundles when KaTeX sources have been changed. I'm not sure how it will impact the performance.

Reply:
`main.js` is pretty small so I can't see there being a huge perf impact when it comes to HMR.